### DISCUSSION TITLE
Is there are chance to export the figures as `csv`?

### DISCUSSION THREAD

**Discussion by Anselmoo** [Data-Centric-AI-Community/fg-data-profiling#984] [Ideas]
Dear community,

thx for help?

I am wondering if there is a chance to export figure-reports like a variable, correlation, or missing_variables as `csv` for rendering in other packages like `seaborn` or `plotly`?

Thx

**Comment by sbrugman** [comment]
@Anselmoo That would be a great feature. At this moment, it is possible to change the plotting method in the package, in order to use seaborn and plotly - simply overriding the methods in [plot.py](https://github.com/ydataai/pandas-profiling/blob/develop/src/pandas_profiling/visualisation/plot.py) will achieve this effect. Supporting alternative plotting backends is on our minds, but not a high priority at the moment.

Even more flexible would be to return the data that is used for plotting in the summary, rather than the serialized plots. This would be a welcome contribution!

**Reply by Anselmoo** [reply]
@sbrugman thx for feedback!
